buildkitd/entrypoint.sh

```diff
@@ -215,12 +215,21 @@ fi
 export TLS_ENABLED
 
 envsubst </etc/buildkitd.toml.template >/etc/buildkitd.toml
+
+# Set up OOM
+OOM_SCORE_ADJ="${BUILDKIT_OOM_SCORE_ADJ:-0}"
+export OOM_SCORE_ADJ
+
+envsubst '${OOM_SCORE_ADJ}' </bin/oom-adjust.sh.template >/bin/oom-adjust.sh
+chmod +x /bin/oom-adjust.sh
```

Comment:
can we `rm oom-adjust.sh.template` at this point?

Reply:
If we want. We dont rm the other template files either though?

Reply:
Doesn't matter anymore, hooks (when we want them) are config options, and the new config value obviates this script.

Still willing to `rm` the other template pieces in here if you want.

Reply:
So I did some real world testing here. I cannot remove any templates if we want to allow the container to restart; otherwise it fails because the files are missing when it attempts to reconstruct any templated file.

Reply:
that really surprises me -- I assumed a restart would also create a new instance of the filesystem, guess we'll need to keep them around. thanks for taking a look.

Reply:
I do wish restarting would remove the backing volume, but it doesnt :(